Makefile.core.mk

```diff
@@ -290,7 +290,7 @@ endif
 
 .PHONY: update-istio
 update-istio: ## Updates the Istio commit hash in the 'latest' entry in versions.yaml to the latest commit in the branch
-	$(eval COMMIT=$(shell yq eval '"git ls-remote " + .latest.repo + ".git | grep " + .latest.branch + " | cut -f 1"' versions.yaml | sh))
+	$(eval COMMIT=$(shell yq eval '"git ls-remote --heads " + .latest.repo + ".git " + .latest.branch + " | cut -f 1"' versions.yaml | sh))
```

Comment:
Please also add `make update-istio` to tools/update_deps.sh so we can get nightly updates for the `latest` version.

Reply:
done